deidentification/archive.py

```diff
@@ -2,34 +2,54 @@
 import tarfile
 import zipfile
 import os
+from deidentification import DeidentificationError
 
 
-def is_archive(filename):
+
+def is_archive_in(filepath):
```

Comment:
What do you think about renaming `is_archive_..`functions to be more explicit about what they doing?
for example :

`is_archive_in`  --> `is_archive_file`
`is_archive_out` --> `is_archive_ext` or `is_archive_extension`

Reply:
I think that a you have a good idea ! I was looking for a name more explicit.